internal/adapters/ffmpeg/manager.go

```diff
@@ -89,7 +91,32 @@ func (m *Streams) addStream(stream domain.Stream) (*ffmpegStream, error) {
 }
 
 func extractClip(clipName string, data []byte) (string, error) {
+	// *** Solução 1 *** -> pegar outputpath porém sem a necessidade de .env
+	currentUser, err := user.Current()
+	if err != nil {
+		fmt.Println("Error getting current user: ", err)
+	}
+	outputPathFile := filepath.Join(currentUser.HomeDir, "Videos", clipName)
```

Comment:
Cara, como esse path sera acessado posteriormente pensei em separar esse codigo em outro lugar:
```suggestion
	outputPath, err := domain.GetOutputPath()
	if err != nil {
		return "", err
	}
	
	outputPathFile := filepath.Join(outputPath, clipName)
```